libs/community/langchain_community/vectorstores/sqlserver.py

```diff
@@ -111,7 +130,28 @@ def __init__(
         self._embedding_store = self._get_embedding_store(table_name, db_schema)
         self._create_table_if_not_exists()
 
+    def _can_connect_with_entra_id(self) -> bool:
+        parsed_url = urlparse(self.connection_string)
```

Comment:
using urlparse will only work if connection_string is something of the form which sqlalchemy expects.. Lets document in the function description this assumption (in addition the PR description)